scripts/run_tests.sh

```diff
@@ -32,21 +32,25 @@ function display_result {
   fi
 }
 
-flake8 --max-complexity 10 --count
-display_result $? 1 "Code style check"
 
-py.test --cov=app --cov-report xml $@ $1
-display_result $? 2 "Unit tests"
+if [ -z "$EQ_FUNCTIONAL_TEST_SUITES" ]; then
+    export EQ_FUNCTIONAL_TEST_SUITES="core"
+fi
 
-# Run front end tests
-npm config set python python2.7
+if [ "$EQ_FUNCTIONAL_TEST_SUITES" = "core" ]; then
```

Comment:
I guess you still get the fail fast indication from the other build, the disadvantage is that you don't free up your travis resource I guess. I am just playing at the moment to see how much time it saves us to see if the cost is worth the benefit